### ISSUE TITLE
JetBrains Gateway: make it clear that you can't use the module on Windows

### ISSUE THREAD

**Issue by phorcys420** [coder/modules#360]
Currently if you use the JetBrains module on Windows, it will not complain about anything and show the button.
If you click it, then JetBrains gateway will load forever and not tell you that Windows support isn't there yet.

I think we should add an assert that the platform is not Windows in the module to make it error instead.

cc: @matifali

**Comment by matifali** [comment]
It does work on Windows. Gateway is available for Windows too. See: https://www.jetbrains.com/remote-development/gateway/
![Image](https://github.com/user-attachments/assets/fe004baa-aa63-418d-bfcd-8900f8dd3f55)

**Comment by matifali** [comment]
I agree, we should add a general filter for modules that are windows only and same for linux only.

**Comment by matifali** [comment]
@bcpeinhardt @Parkreiner is a good candidate for improving the UX of the registry.

We can add a new metadata into the header that can specify the platform where the module can be used.
